pos-print/src/test/java/com/clouway/pos/print/persistent/InMemoryDatastoreRule.kt

```diff
@@ -0,0 +1,26 @@
+package com.clouway.pos.print.persistent
+
+import com.github.fakemongo.Fongo
+import com.mongodb.client.MongoDatabase
+import org.junit.rules.ExternalResource
+
+/**
+ *@author Borislav Gadjev <borislav.gadjev@clouway.com>
+ */
+class InMemoryDatastoreRule : ExternalResource() {
```

Comment:
Why I want to be concrete ? 

If implementation is changed then name have to be changed and all tests will be affected which are using this class.